context-free.md

```diff
@@ -1,21 +1,24 @@
 # Context Free
 
-##  Who is this programming language for?
-
+## Who is this programming language for?
```

Comment:
What do you think is the amount of experience people should be having with graphic design or just design in general when they are using this programming language?